examples/Bunch/include/simulation_defines/param/laserConfig.param

```diff
@@ -176,6 +176,8 @@ const double PULSE_LENGTH_SI = 10.615e-15 / 4.0; //5.0e-15; //25.0e-15 / 1.17741
 /** The laser pulse will be initialized half of PULSE_INIT times of the PULSE_LENGTH before and after the plateau
  *  unit: none */
 const double RAMP_INIT = 20.6146; //25.6415;//SI::aux_calulation::PULSE_TIME_SI/SI::PULSE_LENGTH_SI;
+/* we use a sin(omega*time + laser_phase) function to set up the laser - define phase: */
+const double laser_phase = 0.0;
```

Comment:
pls write it directly uppercase and add the definition range (e.g., periodic in 2 pi)